assets/javascripts/test_result.js

```diff
@@ -586,6 +586,22 @@ function setCurrentPreviewFromStepLinkIfPossible(stepLink) {
   }
 }
 
+function githashToLink(value) {
+  var hashes_regex = /(\b[0-9a-f]{9}\b)/gms;
+  var githashes = value.match(hashes_regex);
+  var repo = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/';
+  var tmpvalues = value;
+  var stat_regex = /(\<a.+?)(?=(\<a))/gms;
+  if (githashes == null) {
+    return;
+  }
+  for (let i = 0; i < githashes.length; i++) {
+    tmpvalues = value.replace(githashes[i], githashes[i].link(repo + githashes[i]));
+    value = tmpvalues;
```

Comment:
```suggestion
    value = value.replace(githashes[i], githashes[i].link(repo + githashes[i]));
```

And remove `var tmpvalues…`.

Reply:
ready for review